### PR TITLE
Add ClamAV to Apply

### PR DESCRIPTION
**Pull request by heidar**
## What does this pull request do?

This PR adds ClamAV to Apply.

## What is the intent behind these changes?

ClamAV is used to scan attachments that are uploaded in the legal aid application process. It runs in a separate container, so it makes sense to have it represented as a separate container on the diagram.

### CODE REVIEW

File: src/main/kotlin/model/Apply.kt [hunk 1]
```diff
@@ -44,12 +45,17 @@ class Apply private constructor() {
...
+      clamAV = system.addContainer("ClamAV", "Clam AntiVirus virus scanner", "ClamAV").apply {
+        CloudPlatform.kubernetes.add(this)
+      }
     }
 
     override fun defineInternalContainerRelationships() {
       web.uses(db, "Connects to")
       sidekiq.uses(queue, "Processes queued jobs from")
       web.uses(queue, "Queues feedback jobs to")
+      web.uses(clamAV, "Scans attachments with")
```
Comment (adrianclay) [inline src/main/kotlin/model/Apply.kt:58]: In the PR description, you refer to scanee as "documents" and here as "attachments".

Just wanted to make sure this was intentional.

**Review by adrianclay — APPROVED** [review]
Outrageous lack of screenshot in the description.

**Review by oliverweighell — APPROVED** [review]
Looks good to me.

File: src/main/kotlin/model/Apply.kt (hunk 1 above)
Comment (heidar) [reply src/main/kotlin/model/Apply.kt:58]: Ah good shout, they can be images actually, so attachments is better.

**Pull request merged**